src/tools/wasm-strip.cc

```diff
@@ -28,6 +28,7 @@ using namespace wabt;
 static std::string s_filename;
 static std::string s_outfile;
 static std::set<std::string_view> v_sections_to_keep{};
+static std::set<std::string_view> v_sections_to_exclude{};
```

Comment:
I would avoid using the term exclude here since it could be interpreted as "exclude from the stripping" i.e. "keep".

I think `remove` is more clear.

Reply:
I agree - I'll rename it!